bucket/wire.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "3.9.2928",
+    "description": "Secure messaging, file sharing, voice calls and video conferences.",
+    "homepage": "https://wire.com/",
+    "license": "GPL-3.0-only",
+    "url": "https://wire-app.wire.com/win/prod/wire-3.9.2928-full.nupkg",
+    "hash": "sha1:b6c5cf28d71511890dac304c1a0817d59171e846",
+    "extract_dir": "lib\\net45",
+    "bin": "Wire.exe",
+    "shortcuts": [
+        [
+            "Wire.exe",
+            "Wire"
+        ]
+    ],
+    "checkver": {
+        "url": "https://wire-app.wire.com/win/prod/RELEASES",
```

Comment:
Are you sure you want to check version fro this? I would try to avoid this after recent Slack updates

Reply:
Well... I thought it's not so bad. Ok, I'll change it to github.